tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/recent_concept_suppression_test.py

```diff
@@ -0,0 +1,216 @@
+"""
+Integration test for covid_ehr_vaccine_concept_suppression module
+
+None
+
+Original Issue: DC1692
+"""
+
+# Python Imports
+import os
+
+# Project imports
+from app_identity import PROJECT_ID
+from cdr_cleaner.cleaning_rules.recent_concept_suppression import RecentConceptSuppression
+from tests.integration_tests.data_steward.cdr_cleaner.cleaning_rules.bigquery_tests_base import BaseTest
+from common import JINJA_ENV, OBSERVATION, DRUG_EXPOSURE, CONCEPT, CONCEPT_RELATIONSHIP, CONCEPT_ANCESTOR
+
+# Third party imports
+from dateutil.parser import parse
+from datetime import date
+from dateutil.relativedelta import relativedelta
+
+
+class RecentConceptSuppressionTest(BaseTest.CleaningRulesTestBase):
+
+    @classmethod
+    def setUpClass(cls):
+        print('**************************************************************')
+        print(cls.__name__)
+        print('**************************************************************')
+
+        super().initialize_class_vars()
+
+        # set the test project identifier
+        project_id = os.environ.get(PROJECT_ID)
+        cls.project_id = project_id
+
+        # set the expected test datasets
+        dataset_id = os.environ.get('COMBINED_DATASET_ID')
+        cls.dataset_id = dataset_id
+        sandbox_id = f"{dataset_id}_sandbox"
+        cls.sandbox_id = sandbox_id
+
+        # Update cutoff_date if necessary. Cutoff date now set to current date
+        cls.cutoff_date = date.today()
+        cutoff_date_str = cls.cutoff_date.isoformat()
+
+        cls.kwargs.update({'cutoff_date': cutoff_date_str})
+
+        cls.rule_instance = RecentConceptSuppression(project_id, dataset_id,
+                                                     sandbox_id)
+
+        cls.vocab_tables = [CONCEPT]
+
+        cls.fq_table_names = [
+            f'{project_id}.{dataset_id}.{CONCEPT}',
+            f'{project_id}.{dataset_id}.{OBSERVATION}',
+            f'{project_id}.{dataset_id}.{DRUG_EXPOSURE}',
+        ]
+
+        cls.fq_sandbox_table_names.extend([
+            f'{cls.project_id}.{cls.sandbox_id}.{cls.rule_instance.sandbox_table_for(OBSERVATION)}',
+            f'{cls.project_id}.{cls.sandbox_id}.{cls.rule_instance.sandbox_table_for(DRUG_EXPOSURE)}',
+            f'{cls.project_id}.{cls.sandbox_id}.{cls.rule_instance.concept_suppression_lookup_table}'
```

Comment:
You should add the `concept_first_use` fully qualified name here also.  Just to make sure the table is cleaned out after the test is completed.